.github/workflows/awstest.yml

```diff
@@ -0,0 +1,30 @@
+name: nf-core AWS test
+# This workflow is triggered on PRs to the master branch.
+# It runs the -profile 'test_full' on AWS batch
+
+on:
+  pull_request:

```

Comment:
should be `push`, but I understand that you just want to test it now.

Reply:
Thanks! Yep. Ill change it in another PR. Just want to see if its working for now 👍